cves/kernel/CVE-2012-2375.yml

```diff
@@ -75,7 +94,8 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs:
+  - 'https://bugzilla.redhat.com/show_bug.cgi?id=822869'
```

Comment:
This might need to just be the code rather than the link.

Reply:
In the description it asked for the Bug ID